### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by asood123**
This pull request adds backers and sponsors from your Open Collective https://opencollective.com/doit ❤️. I'm not familiar with reStructuredText, so please review formatting :). 

It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [more info]. See how it looks on this repo.
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

Thank you for your great contribution to the open source community. You are awesome! 🙌
And welcome to the open collective community! 😊

Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -160,4 +165,52 @@ contributing
...
 
+.. _backers
+
+backers
+-------
+
+
+[Become a backer]((https://opencollective.com/doit#backer)) and show your support to our open source project.
+
+.. image:: https://opencollective.com/webhook/backers.svg?width=890
```
Comment (schettino72) [inline README.rst]: My project is NOT `webhook`.

Comment (schettino72) [reply README.rst]: I can fix this myself. not a big deal. I just realized this might have been manually added.

**Comment by asood123** [comment]
Thanks :). We have an automated script for markdown. I was manually copying
and editing here.

**Comment by schettino72** [comment]
@asood123 , there was a problem with badges on the top. Check the commit above.

Thanks

**Comment by asood123** [comment]
@schettino72 what's the issue? Looks fine to me. It shows zero because no one has donated yet. Once they do, the count will go up.

**Comment by schettino72** [comment]
the link was broken. I have already fixed it. see https://github.com/pydoit/doit/commit/7ec7f4232029e7c0e3ec461757aaca3497a10f78#diff-88b99bb28683bd5b7e3a204826ead112L24

**Comment by asood123** [comment]
ah thank you

**Pull request merged**